### PR TITLE
Exposing root node in visualization app and logging fixes

### PR DESCRIPTION
**Pull request by mbabadi**
A few quick enhancements:
- Exposing root node in the visualization app
- Setting the default root node to "cell" (CL_0000000)
- Print statements to logging

### CODE REVIEW

File: cellarium/cas/settings.py [hunk 1]
```diff
@@ -7,6 +9,8 @@
 MAX_CHUNK_SIZE_SEARCH_METHOD = 500
 CELLARIUM_CLOUD_BACKEND_URL = "https://cellarium-cloud-api.cellarium.ai"
 
+logging.basicConfig(level=logging.INFO, format="* [%(asctime)s.%(msecs)03d] %(message)s", datefmt="%H:%M:%S")
```
Comment (nmalfroy) [inline cellarium/cas/settings.py:12]: Fine for now but you might want to be careful fundamentally changing how logging happens if this gets used in another environment

File: notebooks/quickstart_tutorial.ipynb [hunk 2]
```diff
@@ -334,15 +334,19 @@
...
     "\n",
     "with suppress_stderr():\n",
     "    CASCircularTreePlotUMAPDashApp(\n",
-    "        adata,  # the AnnData file\n",
-    "        cas_ontology_aware_response,  # CAS response\n",
+    "        adata=adata,  # the AnnData file\n",
+    "        root_node=\"CL_0000255\",  # set to CL root node to \"eukaryotic cell\"\n",
     "        cluster_label_obs_column=\"cluster_label\",  # (optional) The .obs column name containing cluster labels \n",
-    "    ).run(port=DASH_SERVER_PORT, debug=False, jupyter_width=\"100%\")"
+    "    ).run(port=DASH_SERVER_PORT, debug=False, jupyter_width=\"1000\")"
```
Comment (nmalfroy) [inline notebooks/quickstart_tutorial.ipynb:349]: I'd leave this as 100%.  This shouldn't be screen specific (especially your gigantic monitor :) )

File: notebooks/quickstart_tutorial.ipynb [hunk 3]
```diff
@@ -232,7 +232,7 @@
    "outputs": [],
    "source": [
     "# Select the annotation model\n",
-    "cas_model_name = '<model_name>'"
+    "cas_model_name = 'cellarium-pca-default-v1'"
```
Comment (nmalfroy) [inline notebooks/quickstart_tutorial.ipynb:235]: I get why you're doing this but keep in mind it's gonna be a pain to maintain this (like if you change the default model at some point and these notebooks are in the wild).  Might make sense to make the APIs take a flag that states to use the default model (or maybe if not model is specified, to just use the default)

**Review by nmalfroy — CHANGES_REQUESTED** [review]
Couple small things

File: cellarium/cas/settings.py (hunk 1 above)
Comment (mbabadi) [reply cellarium/cas/settings.py:12]: Agreed.

File: notebooks/quickstart_tutorial.ipynb (hunk 3 above)
Comment (mbabadi) [reply notebooks/quickstart_tutorial.ipynb:235]: Done.

File: notebooks/quickstart_tutorial.ipynb (hunk 2 above)
Comment (mbabadi) [reply notebooks/quickstart_tutorial.ipynb:349]: Done.

File: cellarium/cas/settings.py (hunk 1 above)
Comment (mbabadi) [reply cellarium/cas/settings.py:12]: Done.

**Comment by mbabadi** [comment]
@nmalfroy back to you!

**Review by nmalfroy — APPROVED** [review]
Looks good! Thanks for making the changes!

**Pull request merged**